geotrellis/tree-prioritization-demo
# Use updated S3 cacheing proxy

This works with the updated S3 cacheing proxy (https://github.com/OpenTreeMap/otm-cloud/pull/294).

Plan for comparative timings on staging:
- Merge this PR and make a 1.1.0 release
- Deploy to staging and gather timings
- If successful, switch `otm-cloud` to provision using 1.1.0
- If not successful, revert these changes and stick with 1.0.0

Testing:
- Check out OpenTreeMap/otm-cloud#294 and reprovision (or recreate) your modeling VM
- Rebuild the modeling JAR (`SJS_HOST=33.33.34.48 SJS_PORT=8090 ./scripts/rebuild.sh`)
- On the modeling VM, `sudo tail -f /var/log/syslog &`
- Turn on a prioritization variable. The syslog output should show first MISSes and then HITs from the cacheing proxy. For example (edited for conciseness):

```
GET /azavea-datahub/catalog/us-census-population-density-30m-epsg3857/8/14406  MISS
GET /azavea-datahub/catalog/us-census-population-density-30m-epsg3857/8/14404  MISS
GET /azavea-datahub/catalog/us-census-population-density-30m-epsg3857/8/14406  HIT
GET /azavea-datahub/catalog/us-census-population-density-30m-epsg3857/8/14404  HIT
```

Connects OpenTreeMap/otm-cloud#278
